### ISSUE TITLE
Bug with `"small paint"` when cursor is after pair

### ISSUE THREAD

**Issue by pokey** [cursorless-dev/cursorless#908]
In the following text:

```
(hello)
```

If the cursor is at the end of the line, saying `"take small paint"` selects `hello`, which is inconsistent with behaviour when the cursor is at the beginning of the line, where it instead selects `(hello)`, as expected.

**Comment by AndreasArvidsson** [comment]
The problem is that we find a matching pair at the end which is incorrect which regards to `requireStrongContainment: true,

**Comment by pokey** [comment]
Yeah fwiw I think #901 is probably caused by the same issue with strong containment
